### PR TITLE
add social share under experiment flag

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@dvoytenko PTAL

/cc @paul-matthews 

we want to put it under experiment since it is not in the validator yet.

**Comment by dvoytenko** [comment]
@erwinmombay What's the validator status?

File: extensions/amp-social-share/0.1/amp-social-share.js [hunk 1]
```diff
@@ -34,6 +42,12 @@ class AmpSocialShare extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    this.isExperimentOn_ = isExperimentOn(this.getWin(), EXPERIMENT);
```
Comment (dvoytenko) [inline extensions/amp-social-share/0.1/amp-social-share.js]: Is experiment already defined in the `tools/experiments.js`? Also what about docs?

Comment (erwinmombay) [inline extensions/amp-social-share/0.1/amp-social-share.js]: i actually didnt add it in tools/experiment.js and thought of adding it to global config? wdyt?

**Comment by erwinmombay** [comment]
@dvoytenko running into problems building the validator internally (there was a bug introduced). will get it submitted by tomorrow hopefully.

File: examples/social-share.amp.html [hunk 2]
```diff
@@ -16,16 +16,16 @@
...
     <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
     <script async src="https://cdn.ampproject.org/v0.js"></script>
   </head>
   <body>
     <h1>Social Share</h1>
 
-    <amp-social-share type="facebook" width="60" height="44" data-text="Hello world" data-url="https://example.com/" data-attribution="AMPhtml">
+    <amp-social-share type="facebook" width="60" height="44" data-text="Hello world" data-url="https://example.com/" data-attribution="AMPhtml" layout=fixed>
```
Comment (dvoytenko) [inline examples/social-share.amp.html]: No need for explicit `layout=fixed` here.

File: extensions/amp-social-share/0.1/amp-social-share.js (hunk 1 above)
Comment (dvoytenko) [inline extensions/amp-social-share/0.1/amp-social-share.js]: What's global config? You mean you want to enable it right away?

**Comment by ericlindley-g** [comment]
@erwinmombay @dvoytenko 
Any updates on this—or is it already in experimental? We'll want to update the docs, too, to avoid confusion

**Comment by dvoytenko** [comment]
I think there are still couple unfinished questions in this PR for @erwinmombay

File: extensions/amp-social-share/0.1/amp-social-share.js (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-social-share/0.1/amp-social-share.js]: added.

File: examples/social-share.amp.html (hunk 2 above)
Comment (erwinmombay) [inline examples/social-share.amp.html]: removed

**Comment by erwinmombay** [comment]
docs updated.

@dvoytenko PTAL

**Comment by erwinmombay** [comment]
@dvoytenko ping, PTAL

File: tools/experiments/experiments.js [hunk 3]
```diff
@@ -50,6 +50,12 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/blob/master/' +
         'README.md#amp-dev-channel',
   },
+  {
+    id: 'amp-social-share',
+    name: 'Amp Social Share',
```
Comment (dvoytenko) [inline tools/experiments/experiments.js]: nit: `AMP` all capitals

**Comment by dvoytenko** [comment]
LGTM with one small nit.

File: tools/experiments/experiments.js (hunk 3 above)
Comment (erwinmombay) [inline tools/experiments/experiments.js]: done.

**Pull request merged**